traffic_ops/app/lib/UI/Server.pm

```diff
@@ -915,20 +916,53 @@ sub readupdate {
 					{
 						$parent_pending{ $rs_servers->single->host_name } = 1;
 					}
+					if (   $prow->reval_pending == 1
+						&& $prow->status->name ne "OFFLINE" )
+					{
+						$parent_reval_pending{ $rs_servers->single->host_name } = 1;
+					}
 				}
 			}
 		}
 	}
 
 	while ( my $row = $rs_servers->next ) {
-		if ( $parent_pending{ $row->host_name } ) {
+		if ( $parent_pending{ $row->host_name } && $parent_reval_pending{ $row->host_name } ) {
```

Comment:
can't we collapse this if/elseif...seems like the only thing that is different is the 

parent_pending => 0|1,
parent_reval_pending => 0|1

maybe a tertiary operator?

parent_pending => (condition) ? 1 : 0,
parent_reval_pending => (condition) ? 1 : 0,